examples/import_movies.rs

```diff
@@ -84,6 +79,14 @@ fn main() {
     println!("Took {:?} in total to insert and build", build);
     println!();
 
+    // Save the database
+    if let Some(path) = args.next() {
+        println!("Save the database in {path}");
+        let path = Path::new(&path);
+        std::fs::create_dir_all(path).unwrap();
+        env.copy_to_file(path.join("data.mdb"), CompactionOption::Enabled).unwrap();
+    }
```

Comment:
Can't we avoid that and directly write to the provided path instead? I want to use your examples to store a lot (and a lot) of vectors, and copying the database if probably not the best thing to do.